src/utils/flow.js

```diff
@@ -3,16 +3,17 @@ import spawn from 'spawndamnit';
 import { Env } from '../Env';
 import findUp from 'find-up';
 import * as path from 'path';
-import { type FlowStatus } from '../types';
+import type { FlowStatus, FlowConfig } from '../types';
+import { Lang } from '../Lang';
 
-async function flow(env: Env, args: Array<string>) {
-  return await spawn(env.flowBinPath, args, { cwd: env.flowRootDir });
+async function flow(path: string, cwd: string, args: Array<string>) {
+  return await spawn(path, args, { cwd });
```

Comment:
I don't understand why any of this needs to change

Reply:
The `Env` class takes as constructor parameters the `flowRootDir` and `flowBinPath`. If we need to call one of these methods to compute either the `flowRootDir` of the `flowBinPath` (which we do to ensure a valid `flowBinPath` exists), we are stuck in a 'requirement' loop.
Changing the method parameters seemed like the best option since the full `Env` is not needed at all, and made the code harder to follow/test.